packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -418,11 +418,16 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                                     // In order to decrease the number of times we have to execute a snapshot refresh,
                                     // if this is the summarizer and we have a cache entry but it is past the defaultSummarizerCacheExpiryTimeout,
                                     // force the network retrieval instead as there might be a more recent snapshot available.
-                                    if (this.hostPolicy.summarizerClient &&
-                                        age > defaultSummarizerCacheExpiryTimeout) {
-                                        // eslint-disable-next-line @typescript-eslint/dot-notation
-                                        props["cacheSummarizerExpired"] = true;
-                                        return undefined;
+                                    // See: https://github.com/microsoft/FluidFramework/issues/8995 for additional information.
+                                    if (this.hostPolicy.summarizerClient) {
+                                        if (age > defaultSummarizerCacheExpiryTimeout) {
+                                            // eslint-disable-next-line @typescript-eslint/dot-notation
+                                            props["cacheSummarizerExpired"] = true;
+                                            return undefined;
+                                        } else {
+                                            // eslint-disable-next-line @typescript-eslint/dot-notation
+                                            props["cacheSummarizerExpired"] = false;
```

Comment:
I am not a fan of adding properties in this way. We should change it to add them similar to how we do it in other places. Maybe also define an interface with the set of properties we want to log. That will make it easier to look for and understand them.

Reply:
That said, do we need to log the false part? If it does not exist, it means the cache didn't expire right?

Reply:
We need to know whether we are a summarizer or not